app/assets/javascripts/controllers/angular-bootstrap/DatepickerCtrl.js

```diff
@@ -0,0 +1,22 @@
+miqAngularApplication.controller('DatepickerCtrl', function ($scope) {
+  $scope.minDate = new Date();
+
+  $scope.disabled = function(date, viewDate, mode) {
+    disabledDate = new Date($scope.$parent.angularForm[viewDate].$viewValue);
+
+    if($scope.$parent.angularForm[viewDate].$untouched) {
+      return ( mode === 'day' && (disabledDate.getUTCDate() === date.getDate()) &&
+                              disabledDate.getUTCMonth() === date.getMonth() &&
+                              disabledDate.getUTCFullYear() === date.getFullYear())
```

Comment:
That is correct, its currently not being used owing to a bug in ```bootstrap datepicker```. 
Apparently if you disable a date, the Angular form flags it as an error.
I do want to leave the ```disabled``` code in there since it might be useful down the line when the bug gets resolved.